aries-site/src/examples/templates/filtering/FilteringTable/FilteringTable.js

```diff
@@ -0,0 +1,121 @@
+import { useContext } from 'react';
+import { Box, DataTable, Heading, ResponsiveContext } from 'grommet';
+import {
+  StatusWarningSmall,
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
+
+import {
+  FilterControls,
+  FiltersProvider,
+  useFilters,
+} from '../../FilterControls';
+import { servers } from './mockData';
+
+export const FilteringTable = () => {
+  // Define which attributes should be made available for the user
+  // to filter upon
+  const filtersConfig = [
+    { property: 'displayName', label: 'Name', filterType: 'CheckBoxGroup' },
+    {
+      property: 'hardware.serialNumber',
+      label: 'Serial',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'state.connected',
+      label: 'State',
+      filterType: 'CheckBoxGroup',
+      render: datum => (datum ? 'Connected' : 'Not Connected'),
+    },
+    {
+      property: 'hardware.powerState',
+      label: 'Power',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'hardware.model',
+      label: 'Model',
+      filterType: 'CheckBoxGroup',
+    },
+  ];
+
+  return (
+    <Box pad="small" gap="medium">
+      <Heading level={2} margin="none">
+        Servers
+      </Heading>
+      <FiltersProvider>
+        <Box gap="medium">
+          <FilterControls
+            data={servers}
+            filters={filtersConfig}
+            searchFilter={{ placeholder: 'Search servers...' }}
+          />
+          <ServerResults />
+        </Box>
+      </FiltersProvider>
+    </Box>
+  );
+};
+
+const columns = [
+  {
+    property: 'hardware.serialNumber',
+    header: 'Serial',
+  },
+  {
+    property: 'state.connected',
+    header: 'State',
+    render: datum => (datum ? 'Connected' : 'Not Connected'),
```

Comment:
Currently this is always true.

```suggestion
    render: datum => (datum.state.connected ? 'Connected' : 'Not Connected'),
```